lib/private/Files/Stream/SeekableHttpStream.php

```diff
@@ -75,8 +75,9 @@ public static function open(callable $callback) {
 
 	/** @var ?resource|closed-resource */
 	private $current;
-	private int $offset = 0;
-	private int $length = 0;
+	private int $offset = 0; // offset of the current chunk
+	private int $length = 0; // length of the current chunk
+	private int $totalSize = 0; // size of the full stream
```

Comment:
```suggestion
	/** @var int $offset offset of the current chunk */
	private int $offset = 0; 
	/** @var int $length length of the current chunk */
	private int $length = 0; 
	/** @var int $totalSize size of the full stream */
	private int $totalSize = 0; 
```

For better documentation and integration in IDEs